scripts/apps/monitoring/views/monitoring-group-header.html

```diff
@@ -27,16 +27,20 @@
         <div class="sortbar" ng-show="customSortOptions">
             <div class="dropdown dropdown--align-right dropdown--hover" dropdown__toggle>
                 <button id="order_button" class="dropdown__toggle" dropdown__toggle>
-                    <span>{{ customSortOptions[customSortOptionActive].label }}</span>
+                    <span ng-if="!customSortOptionActive.field" translate>Default</span>
+                    <span ng-if="customSortOptionActive.field">{{ customSortOptions[customSortOptionActive.field].label }}</span>
                     <span class="dropdown__caret"></span>
                 </button>
                 <ul class="dropdown__menu">
+                    <li>
+                        <button ng-click="selectCustomSortOption(customSortOptionActiveDefault)" translate>Default</button>
```

Comment:
isn't default always `null`? If it is, I'd rather use it inline instead of creating a constant which adds additional lookup step when reading the code, but doesn't add any clarity - inline `null` would be as clear.